### ISSUE TITLE
Error Message says "Ops" instead of "Oops"

### ISSUE THREAD

**Issue by jschroed** [ValveSoftware/steam-for-linux#3030]
The error window that shows up when steam has some trouble running has this message:

"Error"
"Ops! Steam had some trouble to run. That can be a problem with steam itself or with your configuration."

I think it is meant to say "Oops" instead of "Ops".

**Comment by kisak-valve** [comment]
Hello @jschroed, are you able to reproduce this error message on an up to date system?

**Comment by jschroed** [comment]
Sure, I can test this. I'll try to get to it this week. My current hardware now has SteamOS on it but I'll fire up a VM with Arch Linux.

I saw this issue because of issue #3009. Since 3009 is resolved, I'll try to try to force the error message by deleting a file or something. I might even be able to verify both 3009 and this issue.

**Comment by jschroed** [comment]
I finally got a VirtualBox VM running with Arch Linux and Steam but it probably wasn't necessary. I wasn't able to reproduce the issue (I tried renaming a file but Steam re-added it the next time it launched) so I just analyzed the steam launcher script (/usr/bin/steam).

I found a post online that said that the startup script contained the line:
```
zenity --error --text "Ops! Steam had some trouble to run. That can be a problem with steam itself or with your configuration."
```
The latest script doesn't contain the "Ops" text anywhere and the lines with zenity are the "show_message" function and one question with the text "Couldn't find Steam content, did you move it?"

This isn't quite a retest but hopefully it's good enough to close the ticket. I would call the issue resolved for me.
